src/parser.rs

```diff
@@ -136,29 +149,34 @@ impl_rdp! {
         block_start    = _{ variable_start | tag_start | comment_start }
 
         // Actual tags
-        include_tag     = !@{ tag_start ~ ["include"] ~ string ~ tag_end }
-        extends_tag     = !@{ tag_start ~ ["extends"] ~ string ~ tag_end }
-        variable_tag    = !@{ variable_start ~ expression ~ variable_end }
-        comment_tag     = !@{ comment_start ~ (!comment_end ~ any )* ~ comment_end }
-        block_tag       = !@{ tag_start ~ ["block"] ~ identifier ~ tag_end }
-        if_tag          = !@{ tag_start ~ ["if"] ~ expression ~ test? ~ tag_end }
-        elif_tag        = !@{ tag_start ~ ["elif"] ~ expression ~ test? ~ tag_end }
-        else_tag        = !@{ tag_start ~ ["else"] ~ tag_end }
-        for_tag         = !@{ tag_start ~ ["for"] ~ identifier ~ ["in"] ~ idents ~ tag_end }
-        raw_tag         = !@{ tag_start ~ ["raw"] ~ tag_end }
-        endraw_tag      = !@{ tag_start ~ ["endraw"] ~ tag_end }
-        endblock_tag    = !@{ tag_start ~ ["endblock"] ~ identifier ~ tag_end }
-        endif_tag       = !@{ tag_start ~ ["endif"] ~ tag_end }
-        endfor_tag      = !@{ tag_start ~ ["endfor"] ~ tag_end }
+        include_tag      = !@{ tag_start ~ ["include"] ~ string ~ tag_end }
+        import_macro_tag = !@{ tag_start ~ ["import"] ~ string ~ ["as"] ~ simple_ident ~ tag_end}
```

Comment:
Right, but why can't there be a default name, like, the name of the file without the extension(s)?